### PR TITLE
Created API to submit user to database on login

### PR DESCRIPTION
**Pull request by hdinh77**

### CODE REVIEW

File: starter-code/api/api.md [hunk 1]
```diff
@@ -108,3 +108,64 @@ Returns HTTP 200 Success if successfully removed from array.
...
+```
+{
+    "full_name": $String, //Required. 
+    "given_name": $String, //Required.
+    "family_name": $String, //Required.
+    "email": $String, //Required.
+    "drives": $Array, //Required
+    "rides": $Array, //Required.
+    "history": $Array //Required. 
+}
```
Comment (LucienLuc) [inline starter-code/api/api.md:129]: Should probably update this with the new input params.

File: starter-code/api/routes/users.js [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+        });*/
+
+       Users.find({email : body.email}, function (err, docs) {
+            let exists = false;
+            if (docs.length){
+                exists = true;
+            }
+
+            if (!errors.isEmpty() || exists) {
+                return res.status(400).end();
```
Comment (LucienLuc) [inline starter-code/api/routes/users.js:45]:
```suggestion
                return res.status(400).json({errors: errors.array()});
```
So the client can see error messages on faulty inputs.

**Review by LucienLuc — CHANGES_REQUESTED** [review]
Just a couple of small things

**Review by hdinh77 — COMMENTED** [review]
final review

**Pull request merged**